include/bluetooth/audio/audio.h

```diff
@@ -1324,9 +1324,9 @@ struct bt_audio_discover_params {
 	/** Callback function */
 	bt_audio_discover_func_t func;
 	/** Number of capabilities found */
-	uint8_t  num_caps;
+	size_t   num_caps;
 	/** Number of endpoints found */
-	uint8_t  num_eps;
+	size_t   num_eps;
```

Comment:
Which arrays are these referring to? I don't seem them in the same struct?

Reply:
Hmm, this one is a bit unique. 

The struct is part of the callback in 
```c
typedef void (*bt_audio_discover_func_t)(struct bt_conn *conn,
					 struct bt_codec *codec,
					 struct bt_audio_ep *ep,
					 struct bt_audio_discover_params *params);
```

where it reports the number of caps and eps it found, but where the callback is called for each endpoint there is found. So first time it will be called with `num_eps=1`, then 2, then 3, etc. 

I'll revert this one, as that need some other refactoring.